.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,37 @@
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version (optional)'
+        required: false
+        default: 'patch'
+  
+jobs:
+  build:
+    name: Bump Version
+    runs-on: ubuntu-latest
+    steps:
+    - name: ⇣ Checkout
+      uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.GH_BUILDER_TOKEN }}
+
+    - name: Use Node.js 16.10
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.10'
+
+    - name: Run npm version
+      run: |
+        set -e
+        NEW_VERSION="${{ github.event.inputs.version }}"
```

Comment:
but inputs.version is not required

Reply:
Default value is "patch", so if you don't enter anything it'll be running `npm version patch`